build.gradle

```diff
@@ -110,7 +110,7 @@ subprojects {
     ext {
         jacksonVersion = "2.8.11"
         awsJavaSdkVersion = "1.11.63"
-        guavaVersion = "19.0"
+        guavaVersion = "21.0"
```

Comment:
td-client-java uses [guava 21.0](https://github.com/treasure-data/td-client-java/blob/4e45321e89a87e6d975db496871d994dfbb51544/pom.xml#L129-L131), so I also updated guava using in digdag.